tests/Unit/ControlSystem/Actions/Test_GridCenters.cpp

```diff
@@ -55,6 +55,15 @@ struct Metavariables {
   using component_list = tmpl::list<Component<Metavariables>>;
 };
 
+struct UpdateFot {
+  using FoT_t = std::unordered_map<
+      std::string, std::unique_ptr<domain::FunctionsOfTime::FunctionOfTime>>;
+
+  static void apply(gsl::not_null<FoT_t*> current_fots, const FoT_t& new_fots) {
```

Comment:
[optional] We have the `domain::FunctionsOfTimeMap` alias from `FunctionOfTime.hpp`